<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmartPlaylistWizard</class>
 <widget class="QWizard" name="SmartPlaylistWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Smart playlist</string>
  </property>
  <widget class="QWizardPage" name="page_type">
   <property name="styleSheet">
    <string notr="true">QRadioButton {
  font-weight: bold;
}

QLabel {
  margin-bottom: 1em;
}</string>
   </property>
   <property name="title">
    <string>Playlist type</string>
   </property>
   <property name="subTitle">
    <string>A smart playlist is a dynamic list of songs that come from your library.  There are different types of smart playlist that offer different ways of selecting songs.</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="radioButton">
      <property name="text">
       <string>Library search</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Find songs in your library that match the criteria you specify.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="indent">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Last.fm similar artists</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Let Last.fm suggest songs from your library that are similar to one you specify.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="indent">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="SmartPlaylistWizard::SearchPage" name="page_query_search">
   <property name="styleSheet">
    <string notr="true">#termframe {
  border: 1px solid grey;
  border-radius: 10px;
}

#termframe #remove {
background:qlineargradient(x1:0, y1:0, x2:0, y2:1,
                 stop:0 rgba(255,0,0,32), stop: 1 rgba(255,0,0,64));
border: 0px;
border-left: 1px dashed grey;
padding: 6px;
margin-left: 9px;
}</string>
   </property>
   <property name="title">
    <string>Search terms</string>
   </property>
   <property name="subTitle">
    <string>A song will be included in the playlist if it matches all of these conditions.</string>
   </property>
  </widget>
  <widget class="QWizardPage" name="page_query_sort">
   <property name="title">
    <string>Search options</string>
   </property>
   <property name="subTitle">
    <string>Choose how the playlist is sorted and how many songs it will contain.</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Sorting</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QRadioButton" name="sort_random">
         <property name="text">
          <string>Put songs in a random order</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="sort_field">
         <property name="text">
          <string>Sort songs by</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="sort_field_value">
         <item>
          <property name="text">
           <string>Artist A-Z</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Length (shortest first)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Length (longest first)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Year (oldest first)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Year (newest first)</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Limits</string>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0" colspan="2">
        <widget class="QRadioButton" name="radioButton_3">
         <property name="text">
          <string>Show all the songs</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="radioButton_4">
         <property name="text">
          <string>Only show the first</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinBox">
         <property name="suffix">
          <string> songs</string>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>15</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="SmartPlaylistSearchPreview" name="widget" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SmartPlaylistSearchPreview</class>
   <extends>QWidget</extends>
   <header>smartplaylists/smartplaylistsearchpreview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SmartPlaylistWizard::SearchPage</class>
   <extends>QWizardPage</extends>
   <header>smartplaylistwizard.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>sort_field_value</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>sort_field</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>radioButton_4</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
